boards/imix/src/main.rs

```diff
@@ -124,8 +124,13 @@ static mut PROCESSES: [Option<&'static kernel::procs::ProcessType>; NUM_PROCS] =
 #[link_section = ".stack_buffer"]
 pub static mut STACK_MEMORY: [u8; 0x2000] = [0; 0x2000];
 
+#[allow(dead_code)]
```

Comment:
It's probably emblematic of something a bit funkier in the overall pattern we have here, but I think this is better served by removing this and changing line 430 to be `imix.console.start()`. The `dead_code` allow is a bigger hammer that may hide other things in the future